### ISSUE TITLE
Plot method for categorical focal predictors

### ISSUE THREAD

**Issue by strengejacke** [easystats/modelbased#199]
First, it's difficult to see the point estimate, and second, this plot shows no confidence intervals?

``` r
m <- lm(Sepal.Width ~ Petal.Length + Petal.Width + Species, data = iris)
modelbased::estimate_expectation(m, insight::get_datagrid(iris, "Species")) |> plot()
```

![](https://i.imgur.com/mMzSPp4.png)

<sup>Created on 2022-08-11 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by bwiernik** [comment]
There's no error band because the x variable is discrete. With a discrete focal variable, we should make a recipe like that for estimate_means()

**Comment by strengejacke** [comment]
Yes, yes, I know (that's why I added "categorical" in the title), I just wanted to keep this issue in mind, because we have CIs for the predictions, we just don't plot them.

**Comment by bwiernik** [comment]
Yep, this is a bug that has bothered me for a while lol

**Comment by strengejacke** [comment]
same as #145 ?

**Comment by strengejacke** [comment]
Seems to be fixed in #306 

``` r
m <- lm(Sepal.Width ~ Petal.Length + Petal.Width + Species, data = iris)
modelbased::estimate_expectation(m, insight::get_datagrid(iris, "Species")) |> plot()
```

![](https://i.imgur.com/qnfAHjM.png)

<sup>Created on 2025-01-10 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>
